### PR TITLE
FIPS: add specific note for 15 SP5

### PR DESCRIPTION
**Pull request by taroth21**
### PR creator: Description

As discussed in a mail thread with the cert team.

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/security_fips.xml [hunk 1]
```diff
@@ -29,15 +29,23 @@
       <dm:translation>yes</dm:translation>
     </dm:docmanager>
   </info>
+
   <important>
```
[the remark this answers is not in the document]
Comment (denknorr) [reply xml/security_fips.xml:33]: Why should SLEM not apply? it also has cryptography which can be done with FIPS? curl in FIPS

**Review by denknorr — APPROVED** [review]
that's good.

File: xml/security_fips.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (taroth21) [reply xml/security_fips.xml:33]: The question is not only if it generally applies to SLEM or not but also about the product numbers. We build the documentation for SLES, SLED, SLEM and openSUSE  from the same sources within this repository. That's why we generally use entities (variables) for productnames and productnumbers wherever possible, to make sure the productnames and productnumbers  in the generated HTML or PDF output are automatically resolved to what is needed in different contexts. 

But as discussed and agreed with @lvicoun, we can only make limited use of entities for this note, as the content is specific for individual (maintenance) branches anyway must not be backported without making sure it applies to the target branch, too.

[the remark this answers is not in the document]
Comment (taroth21) [reply xml/security_fips.xml:33]: In short: I will merge to `maintenance/SLE15SP5` as it is now - and @lvicoun can still adjust it in the `maintenance/SLE_Micro_5.5` branch later on if needed.

**Comment by taroth21** [comment]
@dariavladykina : Can you please check for any style or language issues before I merge? I'm not sure about the full stop after the reference. Thanks!

File: xml/security_fips.xml [hunk 2]
```diff
@@ -29,15 +29,23 @@
...
+      available in the regular repositories and/or in the
+      <literal>Certifications</literal> module.
     </para>
     <para>
-      For further details, contact your &suse; sales representative.
+      These cryptographic modules, if compatible between both operating
+      systems, are vendor-affirmed, based on the
+      <link xlink:href="https://csrc.nist.gov/csrc/media/Projects/cryptographic-module-validation-program/documents/fips%20140-3/Draft%20FIPS-140-3-CMVP%20Management%20Manual%20v1.2%20%5BDec%2023%202022%5D.pdf"><citetitle>FIPS 140-3 Module
+      Validation Program Management Manual</citetitle></link>, see section
+      <citetitle>7.9.1 Vendor</citetitle>, subitem 1.a i.
```
Comment (dariavladykina) [inline xml/security_fips.xml:48]: I would add a comma here to clarify the bullet point.

Comment (dariavladykina) [inline xml/security_fips.xml:48]:
```suggestion
      <citetitle>7.9.1 Vendor</citetitle>, subitem 1.a, i.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Tanja, please see a suggestion here. Thanks!

**Pull request merged**